### DISCUSSION TITLE
Anschluss am Sensor Wire

### DISCUSSION THREAD

**Discussion by FVerhamme** [dewenni/ESP_Buderus_KM271#88] [General]
Hallo,

könnt Ihr mir bitte erklären, was und wie ich die Anschlüsse am Sensor Wire verwende?
Ich bin leider nicht so fit im lesen von Schaltplänen. Daher helfen mir diese nicht weiter.

FYI: Den J5 habe ich bereits mit einem NTC100 zur Abgasmessung bestückt, aber das ist ja eine andere Geschichte ;-)

Danke und viele Grüße aus dem Westen Deutschlands
Frank

**Comment by the78mole** [comment]
Hallo Frank,

das kommt sehr darauf an, was Du anschließen willst... Der Anschluss ist sehr flexibel. Generell kann man z.B. widerstandsbasierte Sensoren anschließen. Entweder man passt einen der SMD-Widerstände auf dem Board so an, dass er zum Messwiderstand passt oder man macht das extern. Man kann aber auch Spannungen messen. Dafür ist dann aber oft auch ein entsprechender Spannungsteiler notwendig.

Der Anschluss ist eher für die Hardcore-Bastler reserviert :-P Aber wenn Du mir genau beschreibst, was Du messen möchtest, dann helfe ich Dir da natürlich gerne weiter.

Die Frage ist dann vielmehr (weil Du ja hier bei dewenni postest), ob das dann von dewennis Firmware überhaupt unterstützt wird. Aber da kann @dewenni bestimmt weiterhelfen.

Grüße,
Daniel

**Reply by FVerhamme** [reply]
Ich habe hier geschrieben, weil es auf Deiner GitHub Seite keine Möglichkeit dazu gibt ;-)

Ich überlege gerade, was man denn noch so messen könnte. Abgastemp. messe ich ja bereits. Ansonsten fällt mir spontan nichts ein, aber vielleicht Euch.

Gruß Frank

**Reply by Toastbrot20** [reply]
Hallo Frank,
ich messe noch die Rücklauftemperatur und daraus berechne ich die Spreizung. Daraus kann man auch noch die Heizung optimieren. Ansonsten messe ich noch die Raumtemperatur im Heizungskeller.... 
Nur so als Anregung
Viele Grüße
Matthias

**Reply by FVerhamme** [reply]
Hallo Matthias,

wie genau misst Du die Rücklauftemperatur? Fühler einfach ans Rohr geklebt, oder wie? Welchen Sensor nutzt Du?
Und was genau hast Du danach wo optimiert?

Die Raumtemperatur.....hmmmm.....warum eigentlich nicht :-)

Gruß Frank

**Reply by Toastbrot20** [reply]
Ich habe einen gekapselte DS18B20 direkt ans Rohr oberhalb des TemperaturManometer mit Alu-Klebeband unter die Isolierung geklebt. 
Je nach verbauter Hydraulik und Heizungsart sollte die Spreizung ja in einem bestimmten Bereich sein.  Ich konnte meine Umwälzpumpe reduzieren, da Initial die Spreizung nur 2-3Grad hatte. Jetzt habe ich ca.6-10Grad und wenn die Spreizung unter 3Grad geht weiß ich, dass alle Heizkörper aus sind.

**Comment by dewenni** [comment]
Ich habe in der Software eine Unterstützung von 1-2 One-Wire-Sensoren implementiert.
Hier kann man in den Einstellungen die GPIOs vorgeben.
Unterstützt werden derzeit die DS18B20 Sensoren. Diese haben 3-Anschlüsse.
1) VCC
2) GND
3) Sens
Zwischen VCC und Sens wird dann im einfachsten Fall ein 4,7kOhm Widerstand geschaltet und dann wird der "Sens" Anschluss an den konfigurierten GPIO angeschlossen. Welcher GPIO verwendet werden kann, hängt von deinem Board ab.
